edb/pgsql/compiler/dml.py

```diff
@@ -637,6 +637,9 @@ def process_insert_body(
         for shape_el, shape_op in ir_stmt.subject.shape:
             assert shape_op is qlast.ShapeOp.ASSIGN
 
+            if shape_el.path_id.is_linkprop_path():
```

Comment:
What is this new check a result of?

Reply:
Oh, right:
it lets us drop the logic in `_compile_view_shapes_in_set` that checked for linkprop paths and adjusted the path_tip; because previously linkprops got filtered out during an INSERT by inspecting the source. I think we can drop that test now, actually?